apps/builder/app/shared/share-project/share-project.tsx

```diff
@@ -107,16 +117,16 @@ const Menu = ({
           <Item>
             <Label>Name</Label>
             <InputField
-              value={name}
-              onChange={(event) => {
-                onChangeName(event.target.value);
-              }}
+              color={customLinkName.length === 0 ? "error" : undefined}
+              value={customLinkName}
+              onChange={(event) => setCustomLinkName(event.target.value)}
```

Comment:
I would trim it as well
```suggestion
              onChange={(event) => setCustomLinkName(event.target.value.trim())}
```